avro2tf/src/test/scala/com/linkedin/avro2tf/jobs/FeatureListGenerationTest.scala

```diff
@@ -1,28 +1,29 @@
 package com.linkedin.avro2tf.jobs
 
 import java.io.{File, FileOutputStream, PrintWriter}
+import java.nio.charset.StandardCharsets.UTF_8
 
 import scala.collection.mutable
 
 import com.databricks.spark.avro._
 import com.linkedin.avro2tf.parsers.TensorizeInJobParamsParser
-import com.linkedin.avro2tf.utils.Constants.{HASH_INFO, NTV_NAME, NTV_TERM, NTV_VALUE}
+import com.linkedin.avro2tf.utils.Constants.{HASH_INFO, NTV_NAME, NTV_TERM, NTV_VALUE, TMP_FEATURE_LIST}
 import com.linkedin.avro2tf.utils.ConstantsForTest._
 import com.linkedin.avro2tf.utils.WithLocalSparkSession
 
 import org.apache.commons.io.FileUtils
-import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.hadoop.fs.{FileSystem, Path, FileStatus}
 import org.apache.spark.sql.types.{ArrayType, StringType, StructType}
 import org.testng.Assert._
-import org.testng.annotations.Test
+import org.testng.annotations.{DataProvider, Test}
 
 class FeatureListGenerationTest extends WithLocalSparkSession {
 
   /**
    * Test if the Feature List Generation job can finish successfully
    *
    */
-  @Test
+  //@Test
```

Comment:
Why this test is disabled? if yours can cover this then we can remove it.

Reply:
Oh, thanks for catching this. I forgot to uncomment this. The local build test passes with this test included. Will recover the test in the next commit.